docs/documentation/release_notes/topics/24_0_0.adoc

```diff
@@ -195,6 +196,38 @@ spec:
       key: config.xml
 ----
 
+= Keycloak CR resources options
+
+The Keycloak CR now allows for specifying the `resources` options for managing compute resources for the Keycloak container.
+It provides the ability to request and limit resources for the inner Keycloak CR and KeycloakRealmImport CR container.
+
+You can specify your custom values based on your requirements as follows:
+
+[source,yaml]
+----
+apiVersion: k8s.keycloak.org/v2alpha1
+kind: Keycloak
+metadata:
+  name: example-kc
+spec:
+  ...
+  resources:
+    requests:
+      cpu: 1200m
+      memory: 896Mi
+    limits:
+      cpu: 6
+      memory: 3Gi
+----
+
+When the `resources` options are not specified, these default values are used:
+
+* `requests.memory` = 768Mi
+* `limits.memory` = 4Gi
+
+For more details, check the
+https://www.keycloak.org/operator/advanced-configuration[Operator Advanced configuration].
```

Comment:
@vmuzikar Is it sufficient? I'd say so, as it points to the more comprehensive guide.